install/playbooks/roles/borg-backup/tasks/restore-all.yml

```diff
@@ -30,7 +30,10 @@
   when: sogo.install and not sogo_backup_dir.changed
   register: sogo_last_backup_dir
   shell: >-
+    set -o pipefail ;
```

Comment:
Nice catch.  Did ansiblelint flag this?

Reply:
I don't think the semicolons are needed in any of these.

Reply:
Yes, Ansible lint complained about all the missing "pipefail". In fact, there two passing tests that should have failed. Adding the pipefail option revealed the failure.
Thanks.

Reply:
Yes, that's probably true about the semicolon, albeit it doesn't hurt.